site/content/docs/5.3/examples/masonry/index.html

```diff
@@ -57,7 +57,7 @@ <h5 class="card-title">Card title</h5>
       </div>
     </div>
     <div class="col-sm-6 col-lg-4 mb-4">
-      <div class="card bg-dark border-secondary text-center p-3">
+      <div class="card bg-dark border-black text-center p-3">
```

Comment:
Should be moved to https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2369